src/culture/rebels.cpp

```diff
@@ -389,26 +402,13 @@ bool pop_is_compatible_with_rebel_faction(sys::state& state, dcon::pop_id p, dco
 		return false;
 	if(fac.get_type().get_ideology() && fac.get_type().get_ideology_restriction() && fac.get_type().get_ideology() != pop.get_dominant_ideology())
 		return false;
-	if(fac.get_defection_target()) {
-		if(pop.get_is_primary_or_accepted_culture())
-			return false;
-		if(pop.get_culture() == fac.get_defection_target().get_primary_culture())
-			return true;
-		for(auto core : pop.get_province_from_pop_location().get_core()) {
-			if(core.get_identity().get_primary_culture() == pop.get_culture())
-				return false;
-		}
-		return true;
-	}
 	return true;
 }
 
 bool pop_is_compatible_with_rebel_type(sys::state& state, dcon::pop_id p, dcon::rebel_type_id t) {
 	auto fac = fatten(state.world, t);
 	auto pop = fatten(state.world, p);
 
-	if(fac.get_ideology() && fac.get_ideology() != pop.get_dominant_ideology())
```

Comment:
why did you remove this check? This prevents, say, pops that are mostly reactionary from joining communist rebels, etc

Reply:
oh, I see, it just got moved